index.html

```diff
@@ -63,7 +66,7 @@
             /> -->
             <div>
               <button id="goback" class="btn--back">
-                <img src="./src/assets/arrow.png" style="margin-top: 3px" width="15px" height="15px" alt=""/>
+                <img src="./src/assets/arrow.png" style="margin-top: 3px" width="15px" height="15px" alt="form arrow"/>
```

Comment:
I think the alt should clarify the most possible to the user can understand without see the image. For example:

```suggestion
                <img src="./src/assets/arrow.png" style="margin-top: 3px" width="15px" height="15px" alt="Back button icon"/>
```